app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
```

Comment:
- [ ] You are almost there! However, I was not able to create an user (Sign in) or to login, this error is showing up:

<img width="1151" alt="Screen Shot 2022-07-20 at 7 32 21 PM" src="https://user-images.githubusercontent.com/83717285/180105423-8c1dd4a4-78ff-4d1a-8c9b-f16c188664ce.png">

<img width="1137" alt="Screen Shot 2022-07-20 at 7 32 37 PM" src="https://user-images.githubusercontent.com/83717285/180105447-bf596731-7eeb-4684-9244-c735f0f4e827.png">
 
It would be great to fix that in order to test it.

Reply:
- [ ] Lİke the last code reviewer said ı cant also log in and sign in. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/180415469-984b64e8-85ce-49ca-b9ac-460d48d23516.png)

Reply:
This issue is well resolved!  _Signup_ and _login_ working perfectly fine ✔️